src/downloadAzureProject/setupProjectFolder.ts

```diff
@@ -23,7 +24,18 @@ export async function setupProjectFolder(uri: vscode.Uri, vsCodeFilePathUri: vsc
     const resourceId: string = getRequiredQueryParameter(parsedQuery, 'resourceId');
     const devContainerName: string = getRequiredQueryParameter(parsedQuery, 'devcontainer');
     const language: string = getRequiredQueryParameter(parsedQuery, 'language');
+    const node: SlotTreeItemBase | undefined = await ext.tree.findTreeItem(resourceId, { ...context, loadAll: true });
 
+
+    await setupProjectFolderParsed(resourceId, language, vsCodeFilePathUri, context, node, devContainerName);
+}
+
+export async function setupProjectFolderParsed(resourceId: string, language: string,
```

Comment:
Don't do it in this PR (because the diff will be too difficult to read), but eventually we want to move all of the downloadProject logic from this file into `src/commands/downloadProject/downloadProject.ts`. The "commands" folder is where we put most of our code and takes priority over this existing folder/file. This file should basically only be used for uri-related stuff.

Anyways, just keep that in mind as you're setting up the new file structure based on my other comments

Reply:
Ok, I can definitely make this change but I don't understand why it needs to be a different PR